Docs: Web analytics

ivanagas opened the pull request:
## Changes

Adding basic docs for web analytics.

Question:
- Should this be its own section in docs? posthog.com/docs/web-analytics?

## Checklist
- [x] I've added keywords for this page to the rank tracker in Ahrefs
- [x] I've checked the preview build of the article

## Useful resources:

- [The PostHog style guide](https://posthog.com/handbook/growth/marketing/posthog-style-guide)
- [Full list of tags and categories](https://posthog.com/handbook/growth/marketing/tags-and-categories)
- [List of content components](https://posthog.com/handbook/growth/marketing/components)
- [PostHog SEO best practices](https://posthog.com/handbook/growth/marketing/seo-guide)

robbie-c commented on contents/docs/product-analytics/web-analytics.mdx:
```diff
@@ -0,0 +1,53 @@
+    enterprise: partial
+---
+
+> Web analytics are currently in [public beta](/docs/getting-started/enable-betas). This means it's not yet a perfect experience, but we'd love to know your thoughts. Please [share your feedback](http://app.posthog.com/home#supportModal) and [follow our roadmap](https://github.com/PostHog/posthog/issues/18547).
```
I've been calling it opt-in beta elsewhere (to distinguish between now - when people have to opt-in to see it - and the future when it's enabled for most customers but still days beta)

robbie-c reviewed (approved):
LGTM - agree with not mentioning pricing as this is subject to change anyway.

joethreepwood commented on contents/docs/product-analytics/web-analytics.mdx:
```diff
@@ -0,0 +1,53 @@
+
+Getting started is a simple three-step process.
+
+### 1. Install the PostHog on your website
```
```suggestion
### 1. Install PostHog on your website
```

joethreepwood commented on contents/docs/product-analytics/web-analytics.mdx:
```diff
@@ -0,0 +1,53 @@
+
+> Web analytics are currently in [public beta](/docs/getting-started/enable-betas). This means it's not yet a perfect experience, but we'd love to know your thoughts. Please [share your feedback](http://app.posthog.com/home#supportModal) and [follow our roadmap](https://github.com/PostHog/posthog/issues/18547).
+
+Web analytics enables you to easily track and monitor the most important metrics for your website. It provides a simplified dashboard that includes essential metrics like visitors, views, sessions, session duration, bounce rate, paths, referrers, and more.
```
```suggestion
Web analytics enables you to easily track and monitor the many of the most important metrics for your website. Unlike our product analytics tool, web analytics offers a more streamlined and focused experience which is especially suitable for content creators, marketers, or anyone used to tools like Google Analytics. It includes a dashboard that offers essential metrics, such as visitors, views, sessions, session duration, bounce rate, paths, referrers, and more. 
```

joethreepwood commented on contents/docs/product-analytics/web-analytics.mdx:
```diff
@@ -0,0 +1,53 @@
+
+### 1. Install the PostHog on your website
+
+Follow the [installation instructions](/docs/product-analytics/installation) to install either the JavaScript snippet or the JavaScript Web SDK to the website you want to track.
```
```suggestion
Follow the [installation instructions](/docs/product-analytics/installation) to setup either the JavaScript snippet or the JavaScript Web SDK for the website you want to track.
```

joethreepwood commented on contents/docs/product-analytics/web-analytics.mdx:
```diff
@@ -0,0 +1,53 @@
+
+### 2. Enable the web analytics beta
+
+In PostHog, click your profile icon in the top right and then click "Feature preview." In the [feature preview modal](https://app.posthog.com/home#panel=feature-previews), enable "Web Analytics Dashboard"
```
```suggestion
In PostHog, click your profile icon in the top right and then click "Feature preview". In the [feature preview modal](https://app.posthog.com/home#panel=feature-previews), enable "Web Analytics Dashboard".
```

joethreepwood commented on contents/docs/product-analytics/web-analytics.mdx:
```diff
@@ -0,0 +1,53 @@
+
+Like other dashboards in PostHog, the web analytics dashboard is filterable. This means you can filter for data with certain event or person property values. Options include browser, path name, device type, country, and UTMs. Just click the "Add filter" button next to the date range at the top of the dashboard.
+
+This enables you to dive into specific stats for regions, parts of the site, and specific marketing campaigns. 
```
```suggestion
This enables you to dive into specific stats for regions, parts of the site, and specific marketing campaigns. 

For more complex queries, you can still use the product analytics tab as usual.
```

joethreepwood commented on contents/docs/product-analytics/web-analytics.mdx:
```diff
@@ -0,0 +1,53 @@
+
+This enables you to dive into specific stats for regions, parts of the site, and specific marketing campaigns. 
+
+## Frequently asked questions
```
I feel like it's worth adding a question here to cover the difference between web and product analytics, and if one gets you the other, etc. 

Pricing should also be covered. We don't want to make promises, but we should be clear that web analytics is free during the beta period like all other betas - but that users will still pay for the events they send, as normal. 

Finally, we should include a prompt for feedback here.

joethreepwood reviewed (commented):
Some thoughts. Should def be its own section.

ivanagas commented:
@joethreepwood made changes: 
- Moved to `docs/web-analytics`
- Added pricing and difference with product analytics Q&A (with a table)
- Called it an opt-in beta

joethreepwood commented on contents/docs/web-analytics/index.mdx:
```diff
@@ -0,0 +1,79 @@
+
+> Web analytics is currently an [opt-in public beta](/docs/getting-started/enable-betas). This means it's not yet a perfect experience, but we'd love to know your thoughts. Please [share your feedback](http://app.posthog.com/home#supportModal) and [follow our roadmap](https://github.com/PostHog/posthog/issues/18547).
+
+Web analytics enables you to easily track and monitor the many of the most important metrics for your website. 
```
```suggestion
Web analytics enables you to easily track and monitor many of the most important metrics for your website. 
```
